### PR TITLE
Delete test

### PR DESCRIPTION
**Pull request by nirs**
Reproduce bug https://bugzilla.redhat.com/2141978 on minikube and OpenShift.

Notes:
- Test always fails and requires manual cleanup.
- Tested with minikube, should work on openshift.

### CODE REVIEW

**Comment by nirs** [comment]
Example run on OpenShift:

```
$ $src/failover $env -v && $src/delete $env -v
2023-09-07 00:12:58,007 DEBUG   [failover] Parsed arguments: Namespace(verbose=True, name_prefix=None, filename='env.yaml')
2023-09-07 00:12:58,008 DEBUG   [failover] Using environment: {'hub': 'perf1', 'clusters': ['perf2', 'perf3'], 'topology': 'regional-dr'}
2023-09-07 00:12:58,008 DEBUG   [failover] Entering directory '/home/nsoffer/src/ramen/test/delete-test'
2023-09-07 00:12:58,008 INFO    [failover] Failing over to cluster 'perf3'
2023-09-07 00:12:58,733 INFO    [failover] Starting failover for 'drplacementcontrol.ramendr.openshift.io/busybox-sample-placement-1-drpc' to cluster 'perf3'
2023-09-07 00:13:01,146 DEBUG   [failover] drplacementcontrol.ramendr.openshift.io/busybox-sample-placement-1-drpc patched
2023-09-07 00:13:01,872 INFO    [failover] Waiting for 'drplacementcontrol.ramendr.openshift.io/busybox-sample-placement-1-drpc' progression 'Cleaning Up'
2023-09-07 00:13:52,369 DEBUG   [failover] drplacementcontrol.ramendr.openshift.io/busybox-sample-placement-1-drpc condition met
2023-09-07 00:13:52,371 INFO    [failover] Reached failover cleanup progression
2023-09-07 00:13:52,518 DEBUG   [delete] Parsed arguments: Namespace(verbose=True, name_prefix=None, filename='env.yaml')
2023-09-07 00:13:52,518 DEBUG   [delete] Using environment: {'hub': 'perf1', 'clusters': ['perf2', 'perf3'], 'topology': 'regional-dr'}
2023-09-07 00:13:52,519 DEBUG   [delete] Entering directory '/home/nsoffer/src/ramen/test/delete-test'
2023-09-07 00:13:52,519 INFO    [delete] Deleting applcation
2023-09-07 00:13:53,645 INFO    [delete] Deleting subscription.apps.open-cluster-management.io/busybox-sample-subscription-1
2023-09-07 00:13:55,961 DEBUG   [delete] subscription.apps.open-cluster-management.io "busybox-sample-subscription-1" deleted
2023-09-07 00:13:55,964 INFO    [delete] Deleting placement.cluster.open-cluster-management.io/busybox-sample-placement-1
2023-09-07 00:13:59,059 DEBUG   [delete] placement.cluster.open-cluster-management.io "busybox-sample-placement-1" deleted
2023-09-07 00:13:59,061 INFO    [delete] Application was deleted - inspect the clusters manually
```

**Comment by nirs** [comment]
Needs rebase on main

File: test/delete-test/config.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 # SPDX-License-Identifier: Apache-2.0
 
 ---
-repo: https://github.com/ramendr/ocm-ramen-samples.git
-branch: main
+repo: https://github.com/nirs/ocm-ramen-samples.git
+branch: test
```
Comment (nirs) [inline test/delete-test/config.yaml:6]: Not needed now

**Comment by ShyamsundarR** [comment]
Closing as stale, part of general cleanup. Please reopen and update if still required.

**Pull request closed without merging**